gef-ui/package.json

```diff
@@ -22,14 +22,15 @@
     "api-test": "jest --coverage --verbose --forceExit --config=api-test.jest.config.js",
     "build": "rimraf ./public/js && rimraf ./public/css && webpack --mode production --config webpack.prod.config.js",
     "dev": "webpack --config webpack.dev.config.js --watch",
-    "lint": "eslint ./server ./scripts ./component-tests ./test-helpers ./api-tests",
+    "lint": "eslint ./server ./scripts ./component-tests ./test-helpers ./api-tests && npm run type-check",
     "lint:fix": "eslint ./server ./scripts ./component-tests ./test-helpers ./api-tests --fix",
     "start": "npx ts-node server/index.ts",
     "test": "jest --coverage --verbose --config=jest.config.js",
     "test-dev": "jest --verbose --config=jest.config.js",
     "test-file": "jest --config=jest.config.js --verbose --testMatch",
     "test-quick": "jest --config=jest.config.js",
-    "test-watch": "jest --watchAll --coverage --verbose --config=jest.config.js --detectOpenHandles"
+    "test-watch": "jest --watchAll --coverage --verbose --config=jest.config.js --detectOpenHandles",
```

Comment:
Do we need `test-watch` and `test-quick`, `test-file` and `test-dev` scripts?

Reply:
As discussed, these will be reviewed separately @abhi-markan